test/e2e/lib/components/post-editor-toolbar-component.js

```diff
@@ -20,25 +20,11 @@ export default class PostEditorToolbarComponent extends AsyncBaseContainer {
 		this.publishButtonSelector = By.css( '.editor-publish-button' );
 	}
 
-	async ensureSaved( { clickSave = true } = {} ) {
-		const onMobile = driverManager.currentScreenSize() === 'mobile';
-
-		const mobileSaveSelector = By.css(
-			'div.post-editor__content .editor-ground-control__status button.editor-ground-control__save'
-		);
-		const desktopSaveSelector = By.css(
-			'div.card.editor-ground-control .editor-ground-control__status button.editor-ground-control__save'
-		);
-		const saveSelector = onMobile ? mobileSaveSelector : desktopSaveSelector;
-
+	async ensureSaved() {
 		const savedSelector = By.css(
 			'span.editor-ground-control__save-status[data-e2e-status="Saved"]'
 		);
 
-		if ( clickSave === true ) {
```

Comment:
Why do we not need to click the "save selector" any more?

Reply:
Ah, my bad. I missed the fact that `clickSave` attribute's default value is `true` so it's actually doing the click unless told otherwise. Good catch! Will bring it back. (Though all the specs seem to be passing, probably thanks to autosave 😉)

Reply:
Actually, a fun fact I've just discovered - this is working not thanks to the autosave, but because this file isn't used anywhere. 😅 Removed in https://github.com/Automattic/wp-calypso/pull/51410/commits/1677e74bce0e015db59191a44e745627a1aa0e69.

Reply:
🤣 that's funny